lng_tmpl/_base.yml

```diff
@@ -657,7 +657,7 @@ gui_strings:
 - label: LANGUAGE_SIMPL_CHINESE
   string: SIMPLIFIED CHINESE
 - label: OSD_SETTINGS
-  string: OSD settings
+  string: OSD Settings
```

Comment:
This change is not related to pull request.

Reply:
> This change is not related to pull request.

Leave it as is

You don't want another commit just to fix a typo don't you?

Reply:
Yes, ofc its better if you rebase your work without involving additional commits. But if you for some reasons cant do that, we can always go into squash and merge way, sp you can as many commits as you wish.

Reply:
still waiting for undo this